project/ws/app/src/lib/head/left-menu/left-menu.component.html

```diff
@@ -14,28 +14,39 @@
     </a>
```

Comment:
please mention Jira ticket id's in PR title

Reply:
KB-3640, KB-3780, KB-3784 already mentioned